speech/google/cloud/speech/streaming_response.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a GAPIC Speech API response."""
+
+from google.cloud.speech.streaming_result import StreamingSpeechResult
+
+
+class StreamingSpeechResponse(object):
+    """Representation of a Speech API protobuf streaming response.
+
+    :type error: :class:`google.grpc.Status`
+    :param error: Instance of ``Status``
+
+    :type endpointer_type: :class:`~EndpointerType`
+    :param endpointer_type: Enum of endpointer event.
+
+    :type results: list of
+        :class:`google.cloud.speech.v1beta1.StreamingRecognitionResult`
+    :param results: List of protobuf ``StreamingRecognitionResult``.
+
+    :type result_index: int
+    :param result_index: Index for specific result set. Used for updating with
+                         ``interim_results``.
+    """
+    def __init__(self, error=None, endpointer_type=None, results=None,
+                 result_index=None):
+        results = results or []
+        self._error = error
+        self._endpointer_type = EndpointerType.reverse_map.get(
+            endpointer_type, None)
```

Comment:
If we only wanted a string anyway, let's just provide the strings as nice constants that are easy to access?